app/config_files/ParentingAppSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "CoreFiles": "C:/Users/edmun/Code/parenting-app-ui/packages/app-data/sheets",
```

Comment:
Seems like this config file will only work on your machine. Is there a plan to make this more generally usable?

Reply:
Currently this translation process is being managed by @esmeetewinkel using local files, so for now it was though ok to use local file references. But we did also discuss the ideal would be to remove the need for local versions so can consider that as a future improvement

Reply:
Could all of the file paths be relative to the current repo? Most of the file paths that I can see are within the current repo.  I'm happy to assume that if we need to go to another repo that it sits parallel to the current, i.e. to go one level "up" and then nest into the other repo.